app/components/reports/checkbox_input_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div>
   <div class="sci-checkbox-container">
-    <%= check_box_tag @name, true, false, class: 'sci-checkbox' %>
+    <%= check_box_tag @name, true, (@value && @value == 'true'), class: 'sci-checkbox', data: {type: 'CheckboxInputComponent'} %>
```

Comment:
Isn't @value == 'true' just enough here?

Reply:
spaces are missing

Reply:
For the new report, @value is nil. It will generate an error.

Reply:
nil == 'true' will produce error?

Reply:
I will check again